pkg/metadata/sandbox.go

```diff
@@ -55,7 +67,9 @@ type SandboxMetadata struct {
 	// CreatedAt is the created timestamp.
 	CreatedAt int64
 	// NetNS is the network namespace used by the sandbox.
-	NetNS string
+	NetNS *SandboxNetNS
```

Comment:
This is my fault.

Currently, `SandboxMetadata` is designed to be "pure data", it's not designed to hold this kind of data structure, and all update should go through `Update()`.

Let me think about this a little more. We need a better design here.